src/main/java/com/rocketden/main/dao/RoomRepository.java

```diff
@@ -0,0 +1,10 @@
+package com.rocketden.main.dao;
```

Comment:
I ended up moving `ProblemRepository.java` up a folder and deleting the `problem` folder, since I think there's no need to go too modularized (similar to what we decided for the frontend).